test/functional-vanilla.bats

```diff
@@ -151,6 +151,43 @@ spec:
   done
 }
 
+@test "Rename Agents" {
+  initAgents
+  echo "========> Config.yaml before rename"
+  cat ~/.iofog/config.yaml
```

Comment:
All of the rename tests can be handled by a single function defined in functions.bash. Parameters being resource (e.g. agent), old name and new name.

Function should expect empty for `iofogctl get <resource> | grep <oldname>`

Function should expect non-empty string for `iofogctl get <resource> | grep <newname>`